### ISSUE TITLE
Merged vs seqmerge mode

### ISSUE THREAD

**Issue by seppedm** [jtamames/SqueezeMeta#218]
Dear developers,

Currently I'm running a big project (36 transcriptomics samples, 3-7GB each) and I have chosen merged mode as previously discussed elsewhere (using SPAdes). The process is in the merging step for 14 days now (started off with a 16GB total fasta file after merging the assemblies). Our server seems to be handling the minimus2 algorithm fine so far (it's in the "Doing step 20:  Getting overlaps " phase for 10 days now. 

In your manual you state the following about using seqmerge mode: 
"This lightens the computational requirements for the minimus2 algorithm, and can succeed in cases where merged mode would crash."

Does this mean that this mode will only require less resources (less RAM), or is it also meant to be faster than the merged mode? 

Also, if you would happen to have any indication of how long it will take, it would be great, but I assume that's mostly very case-specific. 

Thanks in advance!

**Comment by fpusan** [comment]
Yes, it should require less ram and be faster, but I don't think we can provide an actual estimation. In general minimus is giving us trouble, as execution time seems to scale very badly with assembly size. The seqmerge mode was devised to try and minimize this issue. We have been exploring other options but so far we haven't found a suitable replacement for minimus.

How much RAM memory do you have available? Is running a coassembly out of the question?

**Comment by seppedm** [comment]
Thanks for the answer! Meanwhile I have found some other answers on [this](https://github.com/jtamames/SqueezeMeta/issues/153) thread, which has a very similar set-up. It turns out that even in seqmerge it won't be fast enough to finish in time, so we downscaled the samples to 12 now (for my MSc thesis purposes), which I expect to finish merging today (4 days using 18 threads). 

Server specs: 254GB RAM, 10 cores (but will be upgraded soon). Coassembly is out of the question, for I'm doing metatranscriptomics in the more memory-intensive rnaSPAdes (further discussed [here](https://github.com/ablab/spades/issues/608) and one of the samples couldn't even be assembled on its own!! (We were using SPAdes v3.11.1 which is used in SQM v1.2, solved by using more recent SPAdes version). 

For now everything seems fine for the rest, so I'll close this thread now
